src/vpux_driver_compiler/src/loader/CMakeLists.txt

```diff
@@ -1,115 +1,71 @@
 #
-# Copyright (C) 2022 Intel Corporation.
+# Copyright (C) 2023 Intel Corporation.
 # SPDX-License-Identifier: Apache 2.0
 #
 
-#
-set(CID_COMPONENT "CiD")
-
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/include)
+set(TARGET_NAME VPUXLoader)
 
-set(ELF_DIR "${VPUX_ELF_SOURCE_DIR}/vpux_elf/")
+set(ELF_DIR "${InferenceEngineVPUXPlugin_SOURCE_DIR}/thirdparty/elf/vpux_elf/")
 set(LOADER_DIR_INC "${VPUX_ELF_SOURCE_DIR}/vpux_elf/loader/include/")
-set(LOADER_DIR_HPI_INC "${VPUX_ELF_SOURCE_DIR}/vpux_elf/hpi_component/include/")
-set(LOADER_DIR_HPI_SRC "${VPUX_ELF_SOURCE_DIR}/vpux_elf/hpi_component/src")
-set(LOADER_DIR_SRC "${VPUX_ELF_SOURCE_DIR}/vpux_elf/loader/src")
-set(LOADER_DIR "${VPUX_ELF_SOURCE_DIR}/vpux_elf/loader/")
-
-add_subdirectory(src)
-add_subdirectory(loader)
-
-set(FUNCTIONAL_TARGET vpuxCompilerL0Test)
-set(FUNCTIONAL_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/test/vpux_compiler_l0_tests_common.cpp
-            ${CMAKE_CURRENT_SOURCE_DIR}/test/vpux_compiler_l0_test.cpp
-            ${CMAKE_CURRENT_SOURCE_DIR}/test/vpux_compiler_l0_tests_thread.cpp
-            ${CMAKE_CURRENT_SOURCE_DIR}/test/vpux_compiler_l0_tests_thread2.cpp)
-add_executable(${FUNCTIONAL_TARGET} ${FUNCTIONAL_SOURCES})
-
-add_executable(compilerTest ${CMAKE_CURRENT_SOURCE_DIR}/test/compilerTest.c)
-add_executable(profilingTest ${CMAKE_CURRENT_SOURCE_DIR}/test/profilingTest.c)
-add_executable(loaderTest ${CMAKE_CURRENT_SOURCE_DIR}/test/loaderTest.cpp)
-
-target_link_libraries(compilerTest
-  PUBLIC
-        VPUXCompilerL0
+set(LOADER_DIR_HPI_INC "${VPUX_ELF_SOURCE_DIR/vpux_elf/hpi_component/include/")
```

Comment:
Typo here, please change to ${VPUX_ELF_SOURCE_DIR}

Reply:
Added the fix here: https://github.com/PatrikStepan/npu_plugin/tree/hs/23ww42_snapshot ( on top of this feature branch).
This branch was also validated by @XinWangIntel  and  Jozef Wludzik.
I will create another PR from that branch.